query-parser/src/parse.rs

```diff
@@ -67,6 +67,7 @@ use self::mentat_query::{
     Order,
     OrJoin,
     OrWhereClause,
+    NamedPullAttribute,
```

Comment:
I'd vaguely prefer `Aliased` to `Named`, since that's what we're using throughout, but I'm not hung up on it.

Reply:
I agreed with you, but it turns out that these are not always aliases; this is simply a pull attribute and its name, which sometimes differs from the attribute keyword itself.